src/java/org/streamingpool/core/support/AbstractStreamSupport.java

```diff
@@ -50,6 +51,8 @@
     private ProvidingService providingService;
     @Autowired
     private CreatorProvidingService lazyProvidingService;
+    @Autowired
+    private InstrumentationService dependencyService;
```

Comment:
consider renaming to instrumentationService

Reply:
:+1: